server/gokbg3/src/integration-test/groovy/gokbg3/rest/OrgTestSpec.groovy

```diff
@@ -108,9 +128,13 @@ class OrgTestSpec extends AbstractAuthSpec {
     resp.status == 201 // Created
 
     expect:
-
     resp.json?.name == "TestOrgPost"
     resp.json?._embedded?.ids?.size() == 1
+    resp.json?._embedded?.offices?.size()==3
+    resp.json?._embedded?.offices[0].function.name == "Technical Support"
+    resp.json?._embedded?.offices[2].function.name == "Technical Support"
+    resp.json?._embedded?.offices[1].function.name == "other"
```

Comment:
The order of the returned items in the combo list does not have to be the same as the input. Using `offices*.` should be fine.